src/DefaultCallbackProvider.cpp

```diff
@@ -421,6 +421,10 @@ DefaultCallbackProvider::DefaultCallbackProvider(
                              + "."
                              + region_
                              + CONTROL_PLANE_URI_POSTFIX;
+        // If region is in CN, add CN region uri postfix
+        if(region_.rfind("cn-", 0) == 0) {
```

Comment:
/nit space between `if` and `(`
```suggestion
        if (region_.rfind("cn-", 0) == 0) {
```

Reply:
You're nit picking a cherry-picked commit.

Reply:
This should've been caught by clang format?

Reply:
Leaving this as-is (without the space) will make this `if` statement inconsistent with the other `if` statements above and below it.